web-gateway/conf/messages

```diff
@@ -85,3 +85,6 @@ invalid.duration=Please enter a valid duration
 invalid.units=Please enter a valid unit
 
 createUserInstructions=Here you can create a new user who can participate in and create auctions.
+searchInstructions=Here you can search for auctioned items. Search functionality can be enabled by downloading and running a local instance of elasticsearch. \
+	Steps for doing this can be found in README.md.\
+	You can filter your search by specifying the maximum price of items to search for and the currency to search in.
```

Comment:
Hi @ignasi35 
The newlines are not preserved at the time of rendering in HTML. 
I shall reorder the instructions in the next commit to separate general search instructions and installation instructions.